realm/realm-library/src/objectServer/java/io/realm/RealmAppConfiguration.java

```diff
@@ -273,70 +272,65 @@ public Builder(String appId) {
         }
 
         /**
-         * FIXME
-         *
-         * @param level
-         * @return
-         */
-        public Builder logLevel(int level) {
-            // FIXME: Boundary checks
-            this.logLevel = level;
-            return this;
-        }
-
-        /**
-         * FIXME
+         * Sets the encryption key used to encrypt user meta data. The
```

Comment:
Maybe adding a note that this is _meta data_-only (whatever that covers) and not the actual data with a reference to `SyncConfiguration.Builder.encryptionKey()`

Reply:
Done